src/api/config/routes.rb

```diff
@@ -638,6 +638,9 @@ def self.public_or_about_path?(request)
     end
 
     get '/404' => 'main#notfound'
+
+    get 'public/image_templates' => :index, controller: 'webui/image_templates'
+    resources :image_templates, constraints: cons, only: [:index], controller: 'webui/image_templates'
```

Comment:
Because first one is for the public route! Not sure how you would do it in the style of ``resource``?